sdk/build.gradle

```diff
@@ -0,0 +1,48 @@
+subprojects {
+    apply plugin: 'com.android.library'
+    apply plugin: 'kotlin-android'
+    apply plugin: 'kotlin-android-extensions'
+
+    def ext = rootProject.ext
```

Comment:
Allows us to fetch our dependency and build versions from the `versions.gradle` we declared at the top level build script. We call the extension of the root project to get this information